contrib/docs/tutorials/aws-alb-aad-ocid-auth.md

```diff
@@ -8,24 +8,24 @@ It is assumed an ALB is already serving traffic in front of a Backstage instance
 
 ## Infrastructure setup
 
-### AAD App
+### AAD (Azure Active Directory) App
```

Comment:
I've already introduced AAD earlier, should I repeat?

Reply:
no idea why I missed it